docker-compose.yaml

```diff
@@ -1,19 +1,47 @@
+# Services / profiles:
+#
+#   devrun (default profile) - Build and run the http_server executable.
+#   devrunhw - Same, with SGX_MODE=HW
+#
+#   dev - Start an development environment, without building the project.
+#   devhw - Same, with SGX_MODE=HW
+#
+# (Each service aside from the default is in a profile with the same name.)
+#
+# To use a non-default profile:
+#
+#   docker-compose --profile devrunhw up
+#
+# To use the development environment interactively:
+#
+#   docker-compose build dev
+#   docker-compose run dev
+#
```

Comment:
I added this comment header to help remind me what's what, and to have some quick usages on hand without having to look up docs: does it look sensible?